src/openforms/scss/vendor/_react-select.scss

```diff
@@ -20,5 +20,13 @@
     --border-color: var(--error-fg);
   }
 
-  inline-size: clamp(200px, 100%, var(--of-admin-select-max-inline-size));
+  inline-size: clamp(
+    var(--of-admin-select-min-inline-size),
+    100%,
+    var(--of-admin-select-max-inline-size)
+  );
+
+  .admin-react-select__menu {
+    z-index: 10;
+  }
```

Comment:
magic z-index numbers usually hint at the [stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_positioned_layout/Understanding_z-index/Stacking_context) not being as expected - perhaps there's a better solution?

@stevenbal this may also be what you are running into with the prefill modal

Reply:
I'll take a look if this can be solved in another way 👍

Reply:
This was needed, because the `.submit-row` would be on-top of the react-select menu. The z-index on the `.submit-row` was added because of an issue with the z-index of the tinymce.

I've changed the styling of the `.submit-row` and the tinymce to prevent this stacking issue